### PR TITLE
Clarify wording on borgs NLR rules

### PR DESCRIPTION
**Pull request by SolStar2**
## About the PR
Clarified that borgs may choose whether they forget things upon being removed from a chassis or not.

## Why / Balance
Rules as written imply borgs are prohibited to roleplay forgetting things due to being removed from a chassis. The new wording removes that implication. The Intention of the ruling was clarified in the discord.

## Technical details
Minor change

## Media
N/A

## Requirements

- [X] I have tested all added content and changes.
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes
no

**Changelog**
No cl

### CODE REVIEW

**Comment by SolStar2** [comment]
@Colin-Tel You were the one to provide the clarification so looping you in to check the new wording.

File: Resources/ServerInfo/Guidebook/DeltaV/Rules/GameRules/1_Behave.xml [hunk 1]
```diff
@@ -21,7 +21,7 @@ You must roleplay. Your character's actions, feelings, and knowledge in-game sho
   - Do not engage in meta-communications (i.e. using external channels to communicate with other players in the same game).
 
 ## Rule B1.3: If you die and return to life, follow the new life rules.
-If a player dies, they forget the specific details of their death. [color=#ff0000]Please report players who break this rule.[/color] A borg being removed from its chassis is not a death, and there is no memory loss. 
+If a player dies, they forget the specific details of their death. [color=#ff0000]Please report players who break this rule.[/color] A borg being removed from its chassis is not a death, and may decide there is no memory loss.
```
Comment (Colin-Tel) [inline Resources/ServerInfo/Guidebook/DeltaV/Rules/GameRules/1_Behave.xml]:
```suggestion
If a player dies, they forget the specific details of their death. [color=#ff0000]Please report players who break this rule.[/color] A borg being removed from its chassis is not a death, and may decide for itself whether or not it experiences memory loss. 
```
This is how I'd personally adjust it.

**Review by Colin-Tel — APPROVED** [review]
ShiftySam approves
image

**Comment by SolStar2** [comment]
Now we wait for a heizenbug :)

**Comment by deltanedas** [comment]
need headmin approval still @therealDLondon

**Comment by MilonPL** [comment]
> need headmin approval still @therealDLondon

colin *is* a headmin too

**Comment by deltanedas** [comment]
oh forgor

**Pull request merged**